flatfs.go

```diff
@@ -4,30 +4,32 @@
 package flatfs
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"math"
 	"math/rand"
 	"os"
 	"path/filepath"
-	"strconv"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
-	"github.com/ipfs/go-datastore"
-	"github.com/ipfs/go-datastore/query"
+	"gx/ipfs/QmeiCcJfDW1GJnWUArudsv5rQsihpi4oyddPhdqo3CfX6i/go-datastore"
```

Comment:
Shouldn't be rewritten.

Reply:
Oops! I have been using `git gui` and hand picking the chunks to include, looks like I wasn't careful enough.